src/main/java/net/pms/encoders/TsMuxeRVideo.java

```diff
@@ -159,8 +160,13 @@ public ProcessWrapper launchTranscode(
 		}
 
 		String videoType = "V_MPEG4/ISO/AVC";
-		if (media.getCodecV() != null && media.getCodecV().startsWith("mpeg2")) {
-			videoType = "V_MPEG-2";
+		String codecV = media.getCodecV();
+		if (codecV != null && configuration.isUseMediaInfo()) {
```

Comment:
Why do we need `isUseMediaInfo` here? That seems unrelated to the logic

Reply:
Sorry. Wrong assumption.